pkg/minikube/machine/client.go

```diff
@@ -191,7 +191,7 @@ func (api *LocalClient) Create(h *host.Host) error {
 				// CA cert and client cert should be generated atomically, otherwise might cause bad certificate error.
 				lockErr := api.flock.LockWithTimeout(time.Second * 5)
 				if lockErr != nil {
-					return fmt.Errorf("failed to acquire bootstrap client lock: %v " + lockErr.Error())
+					return fmt.Errorf("failed to acquire bootstrap client lock: %v", lockErr)
```

Comment:
nit: on few places above we're replacing fmt.Error() with errors.New(), but with a format, i think we could use errors.Errorf()

Reply:
The places I'm replacing `fmt.Errorf` with `errors.New` is when there is only a single string without formatting directives to satisfy the linter.

ie.
```
fmt.Errorf(containerdOnlyAddonMsg)

to

errors.New(containerdOnlyAddonMsg)
```